### PR TITLE
Update 15-resourcebasedscheduling

### PR DESCRIPTION
**Pull request by haoqing0110**
Update 15-resourcebasedscheduling

### CODE REVIEW

**Review by qiujian16 — COMMENTED** [review]
thanks, just one thing.

in Graduation Criteria, I think we should add an item for alphat->beta that we need to revisit the API shape before upgrade to beta based on user's feedback

File: enhancements/sig-architecture/15-resourcebasedscheduling/metadata.yaml [hunk 1]
```diff
@@ -8,5 +8,7 @@ approvers:
   - "@qiujian16"
   - "@mdelder"
 creation-date: 2021-08-06
-last-updated: 2021-08-25
-status: provisional
+last-updated: 2021-09-17
+status: implemented
+see-also:
+  - "/enhancements/sig-architecture/15-resourcebasedscheduling/README.md"
```
Comment (qiujian16) [inline enhancements/sig-architecture/15-resourcebasedscheduling/metadata.yaml:14]: point to the original placement proposal https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/6-placements

Comment (haoqing0110) [reply enhancements/sig-architecture/15-resourcebasedscheduling/metadata.yaml]: fixed

**Pull request merged**